core/src/main/java/de/danoeh/antennapod/core/storage/DBWriter.java

```diff
@@ -84,12 +85,15 @@ private DBWriter() {
             if (media != null) {
                 Log.i(TAG, String.format("Requested to delete FeedMedia [id=%d, title=%s, downloaded=%s",
                         media.getId(), media.getEpisodeTitle(), String.valueOf(media.isDownloaded())));
-                boolean result = false;
                 if (media.isDownloaded()) {
                     // delete downloaded media file
                     File mediaFile = new File(media.getFile_url());
                     if (mediaFile.exists()) {
-                        result = mediaFile.delete();
+                        if (!mediaFile.delete()) {
+                            Toast.makeText(context, "Unable to delete file. Rebooting the device could help.",
+                                    Toast.LENGTH_LONG).show();
```

Comment:
Yeah, I know. Never managed to actually trigger this toast, so I should have marked it as WIP